### PR TITLE
[dev] No PBI - Docs updates, Import examples.

### PR DESCRIPTION
**Pull request by War777**
This PR will:

- Update the react-cm-ui package import in examples to:
```

import {
    ...
} from '@saddlebackchurch/react-cm-ui';
```

- Add the `inputMode` prop in the `<Input />` component so mobile browsers can have a better interaction when handling only numbers for example (introduced because is needed for the Giving MVP custom amount input field).

### CODE REVIEW

File: src/surfaces/card/cardBase.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ type PropTypes = {
      * or the new Card component.
      */
     designVersion?: number;
-};
+};}
```
Comment (groberts314) [inline src/surfaces/card/cardBase.tsx]: This doesn't look right

**Review by groberts314 — APPROVED** [review]

File: src/surfaces/card/cardBase.tsx (hunk 1 above)
Comment (War777) [reply src/surfaces/card/cardBase.tsx]: Fixed, thanks for the catch.

**Comment by groberts314** [comment]
@War777 @raoufr Assuming that we want a NPM package release right way (once this is approved and merged), I think 10.6.1 (just bumping patch segment of the semver) is appropriate, as the actual change to the functionality of the main library is pretty modest.

**Comment by raoufr** [comment]
> @War777 @raoufr Assuming that we want a NPM package release right way (once this is approved and merged), I think 10.6.1 (just bumping patch segment of the semver) is appropriate, as the actual change to the functionality of the main library is pretty modest.

Agreed, thank very much @groberts314

**Review by raoufr — APPROVED** [review]
Looking good to me

**Pull request merged**